ignite/cmd/plugin.go

```diff
@@ -376,18 +382,100 @@ plugins:
 	}
 }
 
-func printPlugins() {
-	if len(plugins) == 0 {
-		fmt.Println("No plugin found")
-		return
+func NewPluginDescribe() *cobra.Command {
+	return &cobra.Command{
+		Use:   "describe [path]",
+		Short: "outputs information about the specified plugin. must be registered",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			s := cliui.New(cliui.WithStdout(os.Stdout))
+
+			for _, p := range plugins {
+				if p.Path == args[0] {
+					manifest, err := p.Interface.Manifest()
+					if err != nil {
+						return fmt.Errorf("error while loading plugin manifest\n%s", err)
```

Comment:
errors usually don't have carriage return, and should also wrap (`%w` instead of `%s`) the initial error.

```suggestion
						return fmt.Errorf("error while loading plugin manifest: %w", err)
```